src/out/verify

```diff
@@ -5,6 +5,6 @@
 
 for file in $@; do
     echo "Verifying $file"
-    cbmc -DCBMC --slice-formula --refine --beautify --compact-trace --verbosity 4  $file
+    cbmc -DCBMC --slice-formula --unwind 1000 --refine --compact-trace $file
```

Comment:
is this an acceptable change for everyone?

Reply:
The --unwind change is basically mandatory (CBMC cannot automatically determine the unrolling of any loop where the loop bound depends on a symbolic value, even a bounded one). I'm happy to return --beautify and --verbosity 4 if needed.

Reply:
Yes, please return them.